website/docs/concepts/nodes-networks.md

```diff
@@ -98,6 +98,16 @@ Every network's execution layer works with (and only with) its corresponding "pa
       <td>Sepolia</td>
       <td>Consensus-layer Sepolia is a new network that was created to facilitate testing. The <a href='../install/install-with-script'>Prysm Quickstart</a> shows you how to configure a Merge-ready node on Sepolia. Note that this is a permissioned network, so you can run a node on Sepolia, but not a validator.<br/><br/>This network pair mints and manages <strong>SepplETH</strong>, a type of testnet ETH used exclusively within this network pair.</td>
     </tr>
+    <tr>
+      <td>Holesky</td>
```

Comment:
this is repeated twice?

Reply:
This is desired: there are two columns with the same value.